lib/src/component/abstract_transition.dart

```diff
@@ -281,8 +281,16 @@ abstract class AbstractTransitionComponent<T extends AbstractTransitionProps, S
     }
   }
 
+  var _isMounted = false;
+
+  @override
+  void componentDidMount() {
+    _isMounted = true;
```

Comment:
No that is the `isMounted` method on a React component. They actually recommend this pattern:

"An easy migration strategy for anyone upgrading their code to avoid isMounted() is to track the mounted status yourself. Just set a _isMounted property to true in componentDidMount and set it to false in componentWillUnmount, and use this variable to check your component's status."